Spread envvars docs to related reference sections (#4931)
cocotb/cocotb: docs/source/library_reference_c.rst

GPI Library Reference
*********************

Cocotb contains a native library called :term:`GPI` (Generic Procedural Interface)
that is an abstraction layer for the VPI, VHPI, and FLI simulator interfaces.

.. image:: diagrams/svg/cocotb_overview.svg

The interaction between cocotb's Python and GPI is via a Python extension module called the :ref:`PyGPI <_pygpi>`.

Environment Variables
=====================

.. envvar:: LIBPYTHON_LOC

    The absolute path to the Python library associated with the current Python installation;
    i.e. ``libpython.so`` or ``python.dll`` on Windows.

>     This is determined with ``cocotb-config --libpython`` in cocotb's makefiles.

Reviewer: ```suggestion
    This is determined with ``cocotb-config --libpython`` during build.
```
?
